### ISSUE TITLE
apply codebook to SSA

### ISSUE THREAD

**Issue by rochelle-snyder** [judgelord/correspondence#176]

**Comment by rochelle-snyder** [comment]
Finished a first pass at hand-coding. The vast majority of letters are on behalf of individual constituents, with most of them dealing with various benefit matters, but many of them also deal with things like requests for Social Security card documentation or other matters related to Social Security numbers. 

Only one major coding decision to note: it was often unclear if letters concerning "attorney fees" (there was usually no more detail than this in the subjects) were on behalf of the attorneys or the constituents they were representing, and I also wasn't sure if we wanted to consider attorneys 2s or 1s, so I have coded all of these as type 1/certainty 2/alt type 2. I have done the same with independent contractors when it is clear that the contractor is an individual and not a company.

There was a whole section of logs from 2016 (about 23 pages) that appeared in the PDF but not in the Google sheet, so I converted these pages and added them to the end of the sheet. The same happened with one page of logs from 2012, which I added in by hand in the place where it should have been in the sheet.

Finally, a few terminology items: "N/H" is short for "number holder," and a Numident record is a record of a number holder's original and subsequent applications for Social Security cards.

**Comment by judgelord** [comment]
I agree regarding attorney fees being 1s because they often get paid out of SS settlement, so it seems likely that it is individuals who want SS to pay their attorney so they will stop bothering them. 

I am less clear on contractors--if they are getting paid for providing a service as an LLC would, they may need to be 2s (even though they are not big companies). We may want to try to id a subset of 2s that are (small businesses and individual contractors).
